modules/framework/galasa-parent/dev.galasa.framework.resource.management/src/main/java/dev/galasa/framework/resource/management/internal/DssWatchEventProcessor.java

```diff
@@ -48,7 +48,7 @@ public void run() {
             while(!isDone) {
 
                 // queue.take() blocks momentarily.
```

Comment:
Update/remove this comment now that `take()` isn't being used and `poll()` doesn't block?